flutter/packages/duck_router/lib/src/location.dart

```diff
@@ -68,6 +68,7 @@ typedef LocationBuilder = Widget Function(BuildContext context);
 /// - [DuckPage] for a page that allows defining a custom transition.
 typedef LocationPageBuilder = Page<dynamic> Function(
   BuildContext context,
+  OnPopInvokedCallback onPopInvoked,
```

Comment:
@DelcoigneYves What do you think about this solution (check the latest commit)? 

ps: asking you in the thread to avoid having too many comments in the main PR

Reply:
Looking good to me! I think it still exposes a bit of the internal workings of the DuckRouter, but it's very limited and necessary in this case. 
I would recommend to add some extra information about this property in the typedef documentation, and how and when it should be used. And while we're on it, we could add as well that when providing a custom page, the name property should be filled.
Nice work!

Reply:
Wonderful, yea time to add _all_ the documentation

Reply:
Done!

Reply:
Internal workings removed in #31 👍